### PR TITLE
Adding "time" option to timestamps msg fields in rosbag.py

### PR DESCRIPTION
**Pull request by RoniEmad**
Velodyne lidar ros driver outputs pointcloud msg with timestamps field name "time", so I added it as an option.

### CODE REVIEW

File: src/python/kiss_icp/datasets/rosbag.py [hunk 1]
```diff
@@ -69,12 +69,13 @@ def read_point_cloud(self, bagfile: Path, topic: str, idx: int):
 
         t_field = None
         for field in msg.fields:
-            if field.name == "t" or field.name == "timestamp":
+            if field.name == "t" or field.name == "timestamp" or field.name == "time":
```
Comment (nachovizzo) [inline src/python/kiss_icp/datasets/rosbag.py:72]: Would you mind improving this a bit? :
```py
if field.name in ["t", "timestamp", "time"]:
```

File: src/python/kiss_icp/datasets/rosbag.py [hunk 2]
```diff
@@ -69,12 +69,13 @@ def read_point_cloud(self, bagfile: Path, topic: str, idx: int):
...
                 t_field = field.name
         timestamps = np.ones(points.shape[0])
         if t_field:
             timestamps = np.array(list(self.pc2.read_points(msg, field_names=t_field)))
-            timestamps = timestamps / np.max(timestamps)
```
Comment (nachovizzo) [inline src/python/kiss_icp/datasets/rosbag.py:77]: I'm more into one-liners for these things, but it's personal taste
```py
timestamps = timestamps / np.max(timestamps) if t_field != "time" else timestamps
```

**Review by nachovizzo — CHANGES_REQUESTED** [review]
Thanks for the contributions!

File: src/python/kiss_icp/datasets/rosbag.py (hunk 2 above)
Comment (RoniEmad) [reply src/python/kiss_icp/datasets/rosbag.py:77]: Done c297133565d7aeb557b1a9d549ded93747722ef1

File: src/python/kiss_icp/datasets/rosbag.py (hunk 1 above)
Comment (RoniEmad) [reply src/python/kiss_icp/datasets/rosbag.py]: Done c297133565d7aeb557b1a9d549ded93747722ef1

**Comment by nachovizzo** [comment]
Amazing, thanks @RoniEmad .

FYI: We are currently working on the ROS wrapper ;)

**Comment by RoniEmad** [comment]
Amazing Work.
I was working on the ROS wrapper too :)

**Comment by nachovizzo** [comment]
Great, do you have an open source implementation so we can merge efforts maybe?

I'm pushing the last bis of the python API back to C++ so it's easier to wrap

**Comment by RoniEmad** [comment]
This is my ros wrapper
https://github.com/RoniEmad/kiss-icp/tree/ros_wrapper

It's not complete yet, and needs to be restructured and cleaned.

To use ros, we need to pass the argument --dataloader ros and --topic

It subscribes to the topic and iterate through the pipeline once every call back.
Odometry publisher is not implemented yet.

**Pull request merged**